version: "3.5"
services:
  django:
    labels:
      - "traefik.enable=true"
      - "traefik.docker.network=web"
      - "traefik.http.routers.${CI_ENVIRONMENT_SLUG}-ssl.entryPoints=websecure"
      - "traefik.http.routers.${CI_ENVIRONMENT_SLUG}-ssl.rule=host(`${HOST}`)"
      - "traefik.http.routers.${CI_ENVIRONMENT_SLUG}-ssl.tls=true"
      - "traefik.http.routers.${CI_ENVIRONMENT_SLUG}-ssl.tls.certResolver=leresolver"
      - "traefik.http.routers.${CI_ENVIRONMENT_SLUG}-ssl.service=${CI_ENVIRONMENT_SLUG}-ssl"
      - "traefik.http.services.${CI_ENVIRONMENT_SLUG}-ssl.loadBalancer.server.port=8000"
    networks:
      - default
      - web

networks:
  web:
    external: true
